bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -100,11 +100,11 @@
     private static final Map<String, AddonType> TAG_ADDON_TYPE_MAP = Map.of( //
             "automation", new AddonType("automation", "Automation"), //
             "binding", new AddonType("binding", "Bindings"), //
-            "io", new AddonType("io", "I/O Services"), //
-            "persistence", new AddonType("persistence", "Persistence Services"), //
+            "io", new AddonType("misc", "Misc"), //
```

Comment:
What is the reason for using the "io" tag instead of "misc" @ghys?
IMHO add-ons like "LOG4J2 Extra" would better fit a "misc" tag as "io".

Reply:
It was added here: https://github.com/openhab/openhab-core/blame/main/bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java#L100
when the bundle handlers were expanded to more than bindings.
I believe it was just an honest mistake.

Reply:
If you update the tag I will update the code. :wink:

Reply:
Done.